lib.js

```diff
@@ -0,0 +1,76 @@
+let { readdir, stat } = require("node:fs/promises");
+let path = require("node:path");
+/** @import { FileFinderOptions } from "./types.ts" */
+
+exports.FileFinder = class FileFinder {
+	/**
+	 * @param {string} root
+	 * @param {FileFinderOptions} options?
```

Comment:
I'm not sure it matters, but in the past I had settled on a different syntax:

```suggestion
	 * @param {FileFinderOptions} [options]
```

I believe that seemed more common in the JSDoc context, but I might be wrong; could have been personal preference.

Reply:
Oups, this is not even optional. Fixed in 0f7951c57cec1c12c9b37a094d2c1d1000c64bc1